kolibri/plugins/facility/assets/src/views/FacilityConfigPage/CreateManagementPinModal.vue

```diff
@@ -78,7 +78,7 @@
         context: 'Reminder to sync devices',
       },
       enterPin: {
-        message: 'Enter four numbers to set as your new PIN',
+        message: 'Choose a 4-digit number to set as your new PIN',
```

Comment:
This is identical as the string in the Change PIN modal. Shouldn't we consolidate and put in the `commonCore`?

Reply:
good catch! I'll update